### PR TITLE
Dasd 8441 add schedule query alert template

### PR DESCRIPTION
**Pull request by NickGraham101**

### CODE REVIEW

File: templates/scheduled-query-alert.json [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+            "metadata": {
+                "description": "Whether to trigger the alert based on the total number of breaches or the consecutive number."
+            }
+        },
+        "alertTriggerThreshold": {
+            "type": "int",
+            "metadata": {
+                "description": "The threshold for the trigger."
+            }
+
```
Comment (dale-dfe) [inline templates/scheduled-query-alert.json:66]: Delete space?

File: templates/scheduled-query-alert.json [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+            }
+        },
+        "severity": {
+            "type": "int",
+            "minValue": 0,
+            "maxValue": 4,
+            "metadata": {
+                "description": "The alert severity, used to group alerts by Azure."
+            }
+
```
Comment (dale-dfe) [inline templates/scheduled-query-alert.json:93]: Delete space?

File: templates/scheduled-query-alert.json (hunk 1 above)
Comment (NickGraham101) [reply templates/scheduled-query-alert.json:66]: Deleted

File: templates/scheduled-query-alert.json (hunk 2 above)
Comment (NickGraham101) [reply templates/scheduled-query-alert.json:93]: Deleted

**Review by dale-dfe — APPROVED** [review]
Approved 🚀

**Pull request merged**